shopsite/shop/api/resources.py

```diff
@@ -0,0 +1,51 @@
+from tastypie.resources import ModelResource, ALL, ALL_WITH_RELATIONS
+from tastypie import fields
+from tastypie.authentication import BasicAuthentication
+from tastypie.authorization import DjangoAuthorization
+from tastypie.paginator import Paginator
+from django.contrib.auth.models import User
+from shop.models import Product, Order
+
+class UserResource(ModelResource):
+	class Meta:
+		queryset = User.objects.all()
+		resource_name = "users"
+		fields = ['id', 'username', 'email']
+		authentication = BasicAuthentication()
+		authorization = DjangoAuthorization()
+
+class ProductResource(ModelResource):
+	users = fields.ToManyField(UserResource, "users", full = True)
```

Comment:
Could you please add some python linter to your IDE? pyflakes or something else. and it's good to know about pep8